src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java

```diff
@@ -0,0 +1,176 @@
+package com.dataguise.saas.dto;
+
+import com.google.gson.Gson;
+
+public class ColumnViewUIFormatterDTO {
+
+    String columnName;
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.design.VisibilityModifierCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java L7
    
    Variable &#39;columnName&#39; must be private and have accessor methods.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1580655329>*